.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -0,0 +1,73 @@
+---
+name: Tracking issue
+about: Use this template for upcoming releases.
+title: "[Release Number] Release Checklist"
+---
+
+Target date:
+
+Release Candidate by DD/MM/YYYY
+
+Release by DD/MM/YYYY
+
+Release candidate
+-------------------
+- [ ] Verify that all relevant PRs have been merged to master.
+- [ ] Create a PR against master to bump version number, merge that PR
+- [ ] From the commit just before bumping the version, create a new branch `maint/<release version number>`
+- [ ] Make sure `maint/<release version number>` is included (somehow) in the following block in appveyor.yml 
+```
+branches:
+  only:
+    - ...
+```
+- [ ] Update changelog and open PR targeting a new `maint/<release version number>` branch
+- [ ] Update `ci-src-requirements.txt` if needed
+- [ ] Check MANIFEST and requirements are still up to date.
+- [ ] Update version in setup.py for the prerelease, open a PR against maint/<release version number>
+       - Create a new branch from maint/<release version number>: `git checkout maint/<release version number>; git pull; git checkout -b call-it-anything-you-like`
+       - Set PRERELEASE to "rc1" and IS_RELEASED to true, commit, e.g.git commit -m "Set IS_RELEASED to true for prerelease <release version number>rc1"
+       - Flip IS_RELEASED back to false, commit.
+       - Open a PR against maint/<release version number>
+       - Merge but DO NOT SQUASH
+No squashing so that the tagged commit in the next step will be on the branch (Kit: It is not a big deal even if it was squashed. But some prefer having the commit on the branch.) Alternatively, flip the IS_RELEASED flag in a separate PR.
```

Comment:
```suggestion
No squashing so that the tagged commit in the next step will be on the branch. Alternatively, flip the IS_RELEASED flag in a separate PR.
```

(Not sure if the comment should be kept. I just wanted to remove the reference to myself 😅 )